src/components/FeaturePanel/Climbing/Editor/RouteFloatingMenu.tsx

```diff
@@ -116,6 +116,13 @@ export const RouteFloatingMenu = () => {
             >
               Piton
             </Button>
+            <Button
+              onClick={() => {
+                onPointTypeChange('unfinished');
+              }}
+            >
+              Unfinished
```

Comment:
t() ?

Reply:
What about separate PR for all these translations? There are tons of them.

Reply:
Ok, i wasn't sure.